package.json

```
@@ -14,8 +14,8 @@
     "build:github": "npm i && npm run lint && npm run format:check && npm run test && npm run build:release",
     "build:release": "ng build examples --configuration production",
     "build-storybook": "ng run examples:build-storybook",
-    "test": "ng test --browsers ChromeHeadless --watch false",
-    "test:watch": "ng test",
+    "test": "ng test --browsers ChromeHeadless --watch false --code-coverage",
+    "test:watch": "ng test --code-coverage",
```

I recommend having a separate code coverage command as code coverage takes a LONG time and doesn't (in my experience) add useful information to the regular tests or the "watch" command. You can see the [neo-react package.json](https://github.com/avaya-dux/neo-react-library/blob/main/package.json#L53) as an example. 
```suggestion
    "test": "ng test --browsers ChromeHeadless --watch false",
    "test:watch": "ng test",
    "test:coverage": "ng test --code-coverage",
```